codalab/lib/bundle_cli.py

```diff
@@ -790,6 +790,12 @@ def get_provided_metadata(args):
             help='Operate on this worksheet (%s).' % WORKSHEET_SPEC_FORMAT,
             completer=WorksheetsCompleter,
         ),
+        Commands.Argument(
+            '-m',
+            '--memoize',
+            help='Use a memoized bundle that matches with the given command and dependencies',
```

Comment:
"If a bundle with the same command and dependencies already exists, return it instead of creating a new one."